cross/rackspace-novaclient/Makefile

```diff
@@ -0,0 +1,15 @@
+PKG_NAME = rackspace-novaclient
```

Comment:
If I do not cross compile, the wheel present in the wheelhouse is `rackspace_novaclient-1.5-cp27-none-linux_i686.whl` which is specific to arch i686.
When I cross compile, I get `rackspace_novaclient-1.5-cp27-none-any.whl`.
How could I do without cross compiling it?